index.html

```diff
@@ -30,21 +30,39 @@
   <body>
     <header class="top-menu">
       <a class="toolbarlogo">Alejandro Puente</a>
+
+      <!-- desktop -->
+      <div class="desk-top">
+        <ul class="desk-contact-info small-hide large-show">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <img
+          class="desk-contact-image small-hide large-show"
+          src="./icons/letter.png"
+          alt="Send Mail"
+        />
+      </div>
+
+      <!-- mobile -->
       <img
-        class="toolbar menu"
+        class="toolbar-menu large-hide"
         src="./icons/Enabled.png"
         alt="collapsed menu icon"
       />
     </header>
 
     <section class="welcomeinfo">
-      <h1 class="title">Hey There. I’m Alex!</h1>
-      <h3 class="subtitle">I am a Software Developer</h3>
-      <p class="paragraph">
-        I can help you build a product, feature or website. Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hestiate to contact me.
-      </p>
+      <div class="winfo">
+        <h1 class="title">Hey There. I’m Alex!</h1>
+        <h3 class="subtitle">I am a Software Developer</h3>
+        <p class="paragraph">
+          I can help you build a product, feature or website. Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
```

Comment:
- [ ] I really appreciate your design, you did great work so far, however, please implement these few changes and you will be good to go, see this image below, your `padding-right` value of your `welcomeinfo` section in the desktop version is not the same as on the original design.


![9999999](https://user-images.githubusercontent.com/57562869/160004823-ac4ebd07-de8d-4613-ab44-a5c1027f2f7c.png)

- [ IT SHOULD LOOK LIKE THIS INSTEAD]

![10000000](https://user-images.githubusercontent.com/57562869/160004750-525ed4db-407d-407f-8902-04176c872054.png)

Reply:
thank you for the feedback!
i just adjusted the padding and also the background image to give it more space between elements.